python/pipeline/meso.py

```diff
@@ -416,86 +401,85 @@ def _make_tuples(self, key):
         scan_filename = (experiment.Scan() & key).local_filenames_as_wildcard
         scan = scanreader.read_scan(scan_filename)
 
-        for field_id in range(scan.num_fields):
```

Comment:
same deal as above. Why not all the fields?

Reply:
same answer. Now the populate is ran once per field and not once per scan so no need to loop over all fields